src/apps/content-editor/src/app/components/LockedItem/LockedItem.less

```diff
@@ -1,58 +1,42 @@
 @import "~@zesty-io/core/colors.less";
 @import "~@zesty-io/core/typography.less";
 
-.Wrapper {
-  position: absolute;
-  height: 100%;
-  width: 100%;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  z-index: 100;
-  background-color: rgba(0, 0, 0, 0.5);
-  .Card {
-    background-color: @zesty-white;
-    border: 1px solid @zesty-orange;
-    max-width: 550px;
-    margin-left: -15%;
+.ItemLocked {
+  .ConfirmDialog {
+    background: linear-gradient(@zesty-orange 35%, @appBkgColor 25%);
+    position: relative;
+    overflow: hidden;
+
+    .backgroundIcon {
```

Comment:
@shrunyan 

Can you point me in the right direction im struggling to get the svg icon to overflow:hidden 
as you can see in the image the svg is overlapping some font resulting in a lighter font color. 

<img width="1374" alt="Screen Shot 2021-07-29 at 4 57 59 PM" src="https://user-images.githubusercontent.com/22800749/127580411-6b7396d7-d830-4810-a1c9-c11532cbcd9d.png">

Reply:
Resolved: Added position relative on `ConfirmContent` allowed the content to be on top of svg preventing grayed out text .